src/platform/site-wide/va-footer/containers/Main.jsx

```diff
@@ -278,6 +279,86 @@ export class Main extends React.Component {
             </ul>
           </div>
         </div>
+        <div
```

Comment:
crisis line modal was not ported into the new react footer that's used on team sites- this markup was taken verbatim from our existing html template.